tools/generate-docs

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# Navigate to the docs directory
+cd $(git rev-parse --show-toplevel)/docs
+
+# Make sure dependencies exist
+bundle install
```

Comment:
This requires something like `ruby3.2-rubygem-bundler` to be installed in the execution environment, right?

Reply:
I guess that's why we use the container 😄 

Anyways, that dependency should be documented somewhere if we are going to drop the equivalent from the Dockerfile.

Also this is likely no longer true then: https://github.com/os-autoinst/openQA/blob/db07886e61b59ce822501c5ee439edbc21724e6d/.github/checklist.yml#L7